layouts/partials/loop-person.html

```diff
@@ -6,7 +6,7 @@
 					{{- if (fileExists (printf "static/images/people/%s.jpg" .id )) -}}
 						<img src="{{ (printf "images/people/%s.jpg" .id ) | absURL }}" alt="">
 					{{- else -}}
-						<img src="//www.fillmurray.com/96/96" alt="">
+            <img src="{{ (printf "//ui-avatars.com/api/?name=%s" .name) | absURL }}" alt="">
```

Comment:
Please maintain the same indentation used throughout the layouts. Tabs, not spaces.

No need for `absURL`, since the placeholder avatar would never be hosted locally, but you need to [querify](https://gohugo.io/functions/querify/) the params to ensure they're properly url encoded.

Also, added the proper image size, since the default is 64px.

```suggestion
						<img src="//ui-avatars.com/api/?{{ (querify "name" .name "size" 96) | safeURL }}" alt="">
```